src/GenFacades.Desktop/project.json

```diff
@@ -15,8 +15,5 @@
     },
     "frameworks": {
        "net46": { }
-    },
-    "runtimes": {
-       "win7-x64": { }
```

Comment:
This is an AnyCPU exe and all of it's dependencies are AnyCPU.  At the moment none of the dependency packages require a runtime.  For the most part this is the expectation for the desktop implementations of our packages: it is already implicitly windows only and the assets are only facades which aren't specific to an architecture.  If the shape of the dependency packages changed in the future we'd need to change this project to use a runtime and change how we packaged it (since we'd need to have runtime specific folders that redisted those runtime specific dependencies).